pipx/main.py

```diff
@@ -512,11 +562,11 @@ def setup(args):
     else:
         logging.basicConfig(level=logging.WARNING, format="%(message)s")
 
-    mkdir(constants.PIPX_LOCAL_VENVS)
-    mkdir(constants.LOCAL_BIN_DIR)
-    mkdir(constants.PIPX_VENV_CACHEDIR)
+    mkdir(PIPX_LOCAL_VENVS)
```

Comment:
The tests patch the value of these constants, so they need the `constants.` prefix